test/pytest/test_softmax.py

```diff
@@ -92,7 +89,6 @@ def test_softmax_skipped(backend, io_type):
     assert len(hls_layers) == 2
 
     # Verify hls4ml output is equal to Dense output
-    y_keras = model.predict(X)
-    y_hls4ml = hls_model.predict(X).reshape(y_keras.shape)
-    keras_trace = hls4ml.model.profiling.get_ymodel_keras(model, X)
-    np.testing.assert_allclose(y_hls4ml, keras_trace['dense'], rtol=0, atol=2e-2)
+    y_keras = model.layers[0](X).numpy()  # type: ignore
```

Comment:
Why do you need to do it like this?

Reply:
The original code was redundant (and confusing to me): what is done here is essentially skipping the last layer in Keras. As there are only two, applying the first layer only is enough.
Will revert if you think the original one is better.

Reply:
I think we want just `model.predict(X)`. The section looks like it was cut from a different test. We should use the opportunity to clean up (also the surrounding comments)

Reply:
In this case I shall purge this part, should an equivalent test already exists somewhere.